Jenkinsfile

```diff
@@ -2,7 +2,11 @@
 
 if (JENKINS_URL == 'https://ci.jenkins.io/') {
   buildPlugin(
-    configurations: buildPlugin.recommendedConfigurations().findAll { it.platform == 'linux' },
+    configurations: [
```

Comment:
Without this change, `recommendedConfigurations` forces the Jenkins version to an _older_ jenkins version and a bunch of of things fail.